components/public-api/gitpod/v1/prebuild.proto

```diff
@@ -60,6 +63,17 @@ message WatchPrebuildResponse {
   Prebuild prebuild = 1;
 }
 
+message ListOrganizationPrebuildsRequest {
+  PaginationRequest pagination = 1;
+  string organization_id = 2;
+  Filter filter = 3;
```

Comment:
Move filter inside `ListOrganizationPrebuildsRequest` message if it's not reused anywhere

Reply:
Sounds good, thx